src/util/tools.cc

```diff
@@ -123,11 +123,10 @@ bool startswith(std::string_view str, std::string_view check)
 #endif
 }
 
-std::string toLower(std::string_view str)
+std::string toLower(std::string str)
```

Comment:
This one's somewhat interesting. So you're modifying the function parameter and then copying it. Wouldn't it make more sense to change it to

```
void toLower(std::string& str)
```

Reply:
I added a `toLowerInPlace`, but currently I could only identify a single line where it can be used. But It's in there for the future now.

Reply:
no I mean the toLower makes no sense now. The parameter is std::string, not const ref. So there's a copy.